packages/nextjs/hooks/scaffold-stark/useScaffoldStarkProfile.tsx

```diff
@@ -38,11 +41,16 @@ const fetchProfileFromApi = async (address: string) => {
 
   const profileData = await profileRes.json();
 
+  const id = BigInt(profileData.id).toString();
+
+  const uriRes = await fetch(`${starknetIdApiBaseUrl}/uri?id=${id}`);
+
+  const uriData = await uriRes.json();
```

Comment:
Let's add error handling and log it with `console.error()` similar to the above examples.